### DISCUSSION TITLE
Automatically translate websites?

### DISCUSSION THREAD

**Discussion by kenetik** [ArchiveBox/ArchiveBox#733] [Q&A]
Is it possible to automatically translate websites to a different language when archiving (specifically for screenshots/singlepages)?

**Comment by pirate** [comment]
No, and unlikely to be added. Generally archiving tools try to mutate the content as little as possible during archiving. If you want translations you should translate the page when viewing the archived output, not before you save it.

**Reply by kenetik** [reply]
understood, that's fine. thank you
